README.md

```diff
@@ -146,6 +149,46 @@ spec:
 EOF
 ```
 
+### OpenTelemetry auto-instrumentation injection
+
+The operator can inject and configure OpenTelemetry auto-instrumentation libraries. At this moment, the operator can inject only
+OpenTelemetry [Java auto-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation).
+
+The injection of the Java agent can be enabled by adding an annotation to the namespace, so that all pods within that
+namespace will get the instrumentation, or by adding the annotation to individual PodSpec objects, available as part
+of Deployment, Statefulset, and other resources.
+
+```console
+instrumentation.opentelemetry.io/inject-java: "true"
+```
+
+The value can be 
+* `false` - do not inject
+* `true` - inject
```

Comment:
What happens in this case? If it follows the same resolution mechanism as the sidecar injection, it might make sense to mention here.